04_vsa_kek_extension.sql

```diff
@@ -959,6 +958,8 @@ ALTER TABLE qgep_vl.data_media_kind ADD CONSTRAINT pkey_qgep_vl_data_media_kind_
  ALTER TABLE qgep_od.data_media ADD CONSTRAINT fkey_vl_data_media_kind FOREIGN KEY (kind)
  REFERENCES qgep_vl.data_media_kind (code) MATCH SIMPLE 
  ON UPDATE RESTRICT ON DELETE RESTRICT;
+ALTER TABLE qgep_od.file ADD COLUMN fk_data_media varchar(16);
+ALTER TABLE qgep_od.file ADD CONSTRAINT rel_file_data_media FOREIGN KEY (fk_data_media) REFERENCES qgep_od.data_media(obj_id) ON UPDATE CASCADE ON DELETE set null;
```

Comment:
```suggestion
ALTER TABLE qgep_od.file ADD CONSTRAINT rel_od_file_data_media FOREIGN KEY (fk_data_media) REFERENCES qgep_od.data_media(obj_id) ON UPDATE CASCADE ON DELETE set null;
```

Reply:
The general idea is good, but currently only the fk_dataowner / fk_provider follow this logic. 

The other relations between classes still follow this logic:
ALTER TABLE qgep_od.water_catchment ADD CONSTRAINT rel_water_catchment_aquifier FOREIGN KEY (fk_aquifier) REFERENCES qgep_od.aquifier(obj_id) ON UPDATE CASCADE ON DELETE set null;
https://github.com/QGEP/datamodel/blob/e94fef6296354464e5855be98275596af0b0c3aa/03_qgep_db_dss.sql#L3539

Reply:
Oh I see, sorry I didn't noticed that (I was focused on the table ;)).